build/Publish.targets

```diff
@@ -106,6 +107,13 @@
         <IsDuplicateUpload>true</IsDuplicateUpload>
       </FilesToPublish>
 
+      <!-- Debian installers to be published to nightly feed -->
+      <FilesToPublish Include="$(DependencyAssetsDir)%(InstallerArtifacts.Identity)" Condition=" '%(InstallerArtifacts.InstallerType)' == 'deb' ">
```

Comment:
This adds duplicate entries into the FilesToPublish group for the debian installers. This will cause issues for PublishToTransportFeed and PublishToAzureFeed. 

Other ways to do this: 
 - add the metadata currently in `InstallerArtifacts` to `NativeInstaller` instead.
 - Create a new group `FilesToPublishToDebianFeed` and add this to the `_CheckFilesToPublish` target to ensure these files exist